bin/translate-data.ts

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env ts-node-script
+
+/** Translate all of the files in `./data/` into a single file, `dist/data.json` */
+
+import path from "path";
+import { promises as fs } from "fs";
+import { strict as assert } from "assert";
+
+import toml from "@iarna/toml";
+
+import { assertSchema } from "../src/typeGuards";
+
+const config = {
+  verbose: false,
+};
+
+async function main() {
+  config.verbose == "-v" in process.argv;
```

Comment:
Should this be a single `=`?

Reply:
Huh. I remember using the verbose flag, I wonder how that happened.